File: pset1/cash/cash.c
//Takes cash value and gives minimum number of coins to get amount
#include <stdio.h>
#include <cs50.h>
#include <math.h>



int main(void)
{
    int m;
    float n;
    int o = 0;
    do
    {
        n = get_float("Change owed: ");
    }
    while (n <= 0);
    m = round(n * 100);
    while (m - 25 >= 0)
    {
        m = m - 25;
        o++;
    }
    while (m - 10 >= 0)
    {
        m = m - 10;
        o++;
    }
    while (m - 5 >= 0)
    {
        m = m - 5;
        o++;
    }
    while (m - 1 >= 0)
    {
        m = m - 1;
        o++;
    }
    printf("%i\n", o);
}

File: pset6/cash/cash.py
# Takes cash value and gives minimum number of coins to get amount
from cs50 import get_float

while True:
    n = get_float("Change owed: ") * 100
    if n >= 0:
        break

o = 0

while n - 25 >= 0:
    n -= 25
    o += 1

while n - 10 >= 0:
    n -= 10
    o += 1

while n - 5 >= 0:
    n -= 5
    o += 1

while n - 1 >= 0:
    n -= 1
    o += 1

print(o)